core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2737,53 +2737,90 @@ public String toString() {
           .setScope(Scope.MASTER)
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .build();
-  // In Java8 in container environment Runtime.availableProcessors() always returns 1,
-  // which is not the actual number of cpus, so we set a safe default value 8.
-  public static final PropertyKey MASTER_RPC_EXECUTOR_PARALLELISM =
```

Comment:
can we keep these around and set the default FJP or TPE parallelism to this for better back compat?

Reply:
I set aliases for old FJP related settings. However they won't take effect due to switching to TPE by default.
Rest are untouched already (core/max pool etc...)

Beyond that, any custom configuration that touches parallelism is most probably wrong.